packages/server/src/utils/logger.js

```diff
@@ -0,0 +1,43 @@
+import pino from 'pino'
+import { merge } from '@ditojs/utils/lib/object/merge'
+import { nanoid } from 'nanoid'
+
+export function createLogger({
+  level = 'info',
+  prettyPrint = prettyPrintConfig,
+  serializers = {}
+} = {}) {
+  return pino({
+    level,
+    serializers: merge({}, serializers, {
+      err: pino.stdSerializers.err,
+      req: pino.stdSerializers.req,
+      res: pino.stdSerializers.res,
+      // Only id, to not inadvertently log PII.
+      user: user => ({ id: user.id })
+    }),
```

Comment:
@ptoussai I only noticed this after merging, but what's the reason for using `merge()` here instead of this?

```js
{
  ...serializers, 
  err: pino.stdSerializers.err,
  req: pino.stdSerializers.req,
  res: pino.stdSerializers.res,
  // Only id, to not inadvertently log PII.
  user: user => ({ id: user.id })
}
```

I can change it of course but I'm wondering if I'm overlooking something?

Reply:
Also, should we not allow `serializers` to override the standard ones, instead of the other way round?

Reply:
I would suggest this:
```js
    serializers: {
      ...pino.stdSerializers,
      // Only id, to not inadvertently log PII.
      user: user => ({ id: user.id }),
      ...serializers
    },
```

Reply:
or this:
```js
  const { err, req, res } = pino.stdSerializers
  // Only include `id` from the user, to not inadvertently log PII.
  const user = user => ({ id: user.id })
  return pino({
    level,
    serializers: {
      err,
      req,
      res,
      user,
      ...serializers
    },
```

Reply:
Yes this works !